data-prepper-plugins/parse-json-processor/src/main/java/org/opensearch/dataprepper/plugins/codec/json/NewlineDelimitedOutputCodec.java

```diff
@@ -19,7 +19,7 @@
  * An implementation of {@link OutputCodec} which deserializes Data-Prepper events
  * and writes them to Output Stream as ND-JSON data
  */
-@DataPrepperPlugin(name = "newline", pluginType = OutputCodec.class, pluginConfigurationType = NewlineDelimitedOutputConfig.class)
+@DataPrepperPlugin(name = "ndjson", pluginType = OutputCodec.class, pluginConfigurationType = NewlineDelimitedOutputConfig.class)
 public class NewlineDelimitedOutputCodec implements OutputCodec {
```

Comment:
Please also rename this class to `NdjsonOutputCodec`. Thanks!

Reply:
Hi @dlvenable , As suggested we have renamed the classes and all necessary changes have been implemented throughout the codebase.